### PR TITLE
miscellaneous(pr-template): add PR template, release drafter and mergeable

### PR DESCRIPTION
**Pull request by sidMulay**
Jira: [BB-3380](https://hello-haptik.atlassian.net/browse/BB-3380)

### Changelog

- Add `mergeable.yml`, `pull_request_template.md` and `release-drafter.yml`

### CODE REVIEW

**Review by ranvijayj — APPROVED** [review]
LGTM

**Review by ranvijayj — APPROVED** [review]
LGTM

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    label: enhancement
+  - title: 🧪 Tests
+    label: test
+  - title: 📎 Dependencies
+    label: dependencies
+  - title: 👺 Miscellaneous 
+    label: miscellaneous
+exclude-labels:
+  - miscellaneous
+  - release
```
Comment (swapnilmisal) [inline .github/release-drafter.yml:24]: add a new line here... please @sidMulay

File: .github/mergeable.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        regex: '(do-not-merge)'
+        message: 'This PR is work in progress. Cannot be merged yet.'
+
+    description:
+      no_empty:
+        enabled: true
+        message: 'Can you please add a description!'
+      must_exclude:
+        regex: 'do not merge'
+        message: 'This PR is work in progress. Cannot be merged yet.'
```
Comment (swapnilmisal) [inline .github/mergeable.yml:21]: add a new line here too please.... @sidMulay

**Review by swapnilmisal — CHANGES_REQUESTED** [review]

**Review by swapnilmisal — APPROVED** [review]

**Pull request merged**